packages/cli/src/commands/upgrade/upgrade.js

```diff
@@ -163,6 +164,20 @@ const installDeps = async (newVersion, projectDir) => {
   }
 };
 
+const installCocoaPodsDeps = async (projectDir, thirdPartyIOSDeps) => {
```

Comment:
this step is likely to fail, e.g. because of conflicts in Podfile. Can we wrap it in try/catch and output a helpful message?

Reply:
done, not sure if more detailed message is necessary here?